index.html

```diff
@@ -13,8 +13,8 @@
 				<a href="index.html"> <h1>Need Games</h1> </a>
 			</div>
 			<input type="search" class="text-field" />
-			<div class="navbar-cta"
-				><a href="./screens/login-page/login-page.html" class="btn btn-link">Hello, Login</a>
+			<div class="navbar-cta">
+				<a onclick="openModal('login-modal')" id="login-text" class="btn btn-link">Hello, Login</a>
```

Comment:
suggestion: don’t use onclick in html. implement this on the js side.

Reply:
+1 It creates problem